process.rb

```diff
@@ -83,6 +83,7 @@ def slugify(word)
   office_elections_by_label = office_elections.group_by(&:label)
   election_content = YAML.dump(
     'title' => election[:title],
+    'data_key' => election[:name],
```

Comment:
What if it was something a little more specifically descriptive? 'election_id'?

Reply:
wfm